draft-add-requirements.md

```diff
@@ -32,89 +56,180 @@ informative:
 
-## Discover resolvers associated with a manually-configured resolver
+## Client-selected resolvers {#client-selected}
+
+Client devices often allow the device administrator to select a specific DNS resolver to use
+on certain networks, or on all networks. Historically, this selection was specified only with an
+IP address.
+
+Discovering which equivalent encrypted resolvers are offered by the same entity allows the client
+to "upgrade" connections to use encrypted DNS.
+This can provide several benefits:
+
+- Prevent devices along the network path to the selected resolver from observing client DNS messages
+- Verify that answers come from the selected DNS resolver
+- Authenticate that the DNS resolver is the one selected by the client
+
+In doing so it is critical that the new resolver is an equivalent resolver. Switching to a non-equivalent
+alternative resolver would break the expectation of the user who previously selected that resolver.
 
-A client with manual configuration directing it to use a particular resolver will generally use that resolver. However if that communication is
-unencrypted, it is beneficial for the client to learn which associated resolvers are offered by the same entity and provide the same semantics.
 
 | Requirement | Description |
 | R5.1 | Example requirement |
 
-## Discover resolvers associated with a VPN-configurated resolver
+## VPN resolvers {#vpn}
+
+Virtual Private Networks (VPNs) also can provision DNS resolvers. In addition to being able to use
+DHCP or RAs, VPNs can provision DNS information in an explicit configuration message. For example,
+IKEv2 can provision DNS servers using Configuration Attributes {{?RFC7296}}.
+
+VPNs can also configure Split DNS rules to limit the use of the configured resolvers to specific domain
+names {{?RFC8598}}.
+
+Discovering an encrypted resolver that is provisioned by a VPN can provide the same benefits
+as doing so for a local network, but applied to the private network. When using Split DNS, it becomes
+possible to use one encrypted resolver for private domains, and another for other domains.
 
-Narrative required.
 
 | Requirement | Description |
 | R6.1 | Example requirement |
 
 
 # Discovery of limited domain resolvers
 
-Narrative required.
+Similar to how VPN DNS configurations can use Split DNS for private names, other network environments
+can support resolution of names that are specific to the local environment. For example, an enterprise-managed
+Wi-Fi network might be able to access both the Internet and a private intranet. In such a scenario, the private
+domains managed by the enterprise might only be resolvable using a specific DNS resolver.
+
+Discovering an encrypted resolver for a subset of names allows a client to perform Split DNS while maintaining
+the benefits of encrypted DNS. For example, a client could use a client-selected encrypted resolver for most
+domains, but use a different encrypted resolver for enterprise-private domains.
```

Comment:
I suggest replace "most domains" with "public domains".

Reply:
Yes, this makes the example clearer. Done.